### ISSUE TITLE
Could you kindly share how to train new model?

### ISSUE THREAD

**Issue by a-ejiri** [6gsn/marine#20]
Hi!
I appreciate for sharing your marine. I used your pretrained model trained by JSUT.
Though pretrained model works very well in ordinary corpus, when I used it in my original corpus (corpus about games or animations), its accuracy was about 50%.
So I wanted to build my model by using my corpus set and testify accuracy.

I know you described it will come soon. I'm very looking forward to it, and want to know when you plan to share it.
Thank you.

**Comment by 6gsn** [comment]
Hi, @a-ejiri!
Thank you for your interest and comments! Here are my answers to your comments;

> I used your pretrained model trained by JSUT. Though pretrained model works very well in ordinary corpus, when I used it in my original corpus (corpus about games or animations), its accuracy was about 50%.

That's a pretty predictable result! As you know, the JSUT corpus covers all of the main pronunciations of daily-use Japanese characters [Sonobe, 2017]. However, your test set seems to include a lot of named entities regarding specific domains (e.g., game, animation). Hence, the current pre-trained model may not cover the samples without fine-tuning.

I'm working on preparing the documents and the recipes for building a model with our codes! And the jobs will be finished in serval weeks. I would appreciate your patience in a few whiles!

Thanks!

[Sonobe, 2017] Ryosuke Sonobe, Shinnosuke Takamichi and Hiroshi Saruwatari,  "[JSUT corpus: free large-scale Japanese speech corpus for end-to-end speech synthesis](https://www.google.com/url?q=https%3A%2F%2Farxiv.org%2Fabs%2F1711.00354&sa=D&sntz=1&usg=AOvVaw0Mp0MM8n9vnQv61wjBabRH)," arXiv preprint, 1711.00354, 2017.
